syntax = "proto3";

package KVSTransactions;

service KeyValueStore {
  rpc GetCommand (GetRequest) returns (GetReply) {}
  rpc PutCommand (PutRequest) returns (PutReply) {}
}

message GetRequest {
  string key = 1;
}

message PutRequest {
  string key = 1;
  string value = 2;
}

message GetReply {
  string value = 1;
}

message PutReply {
  string status = 1;
}